ui/tx/assetFlows/components/NovesActionSnippet.tsx

```diff
@@ -0,0 +1,115 @@
+import { Box, Hide, Popover, PopoverArrow, PopoverContent, PopoverTrigger, Show, Skeleton, Text, useColorModeValue } from '@chakra-ui/react';
+import type { FC } from 'react';
+import React from 'react';
+
+import TokenEntity from 'ui/shared/entities/token/TokenEntity';
+import IconSvg from 'ui/shared/IconSvg';
+
+import type { NovesFlowViewItem } from '../utils/generateFlowViewData';
+import NovesTokenTooltipContent from './NovesTokenTooltipContent';
+
+interface Props {
+  item: NovesFlowViewItem;
+  isLoaded: boolean;
+}
+
+const NovesActionSnippet: FC<Props> = ({ item, isLoaded }) => {
+  const popoverBg = useColorModeValue('gray.700', 'gray.300');
```

Comment:
Has our designer agreed to use a custom background for the tooltip?

Reply:
I was under the impression that yes